index.html

```diff
@@ -9,6 +9,31 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class= "cover-text-container">
+    <img class= "cover-page" 
+    src="https://trello-attachments.s3.amazonaws.com/607457e8ad4162814da41ef0/607457e8ad4162814da41f17/x/6554b96f839abbe659429a9f8c79115d/book-cover.png" 
+    alt="cover"
```

Comment:
The `alt` should be a short text that describes what the image is showing if for example it weren't there.
For more info about this, [check this out](https://support.siteimprove.com/hc/en-gb/articles/115000013031-Accessibility-Image-Alt-text-best-practices)